docs/howtos/vap-migration.md

```diff
@@ -0,0 +1,161 @@
+---
+sidebar_label: ValidatingAdmissionPolicy migration
+sidebar_position: 35
+title: ValidatingAdmissionPolicy migration
+description: Discusses how to migrate from Kubernetes VAP policies to Kubewarden.
+keywords: [kubewarden, kubernetes, cel, vap, validatingadmissionpolicy]
+doc-persona: [kubewarden-user, kubewarden-operator, kubewarden-distributor, kubewarden-integrator]
+doc-type: [howto]
+doc-topic: [vap-migration]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/howtos/vap-migration"/>
+</head>
+
+Starting from Kubernetes v1.26, the [ValidatingAdmissionPolicy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/)
+provides a way to write custom admission policies in Kubernetes. The policies are
+written with the [Common Expression Language (CEL)](https://cel.dev/), extended with
+some [Kubernetes-specific extensions](https://kubernetes.io/docs/reference/using-api/cel/).
+ValidatingAdmissionPolicy reached stability in Kubernetes v1.30.
+
+Kubewarden provides a CEL policy that is capable of running Kubernetes VAP policies without any modifications.
+You can read more about the CEL policy in [this section](../tutorials/writing-policies/CEL/01-intro-cel.md) of Kubewarden's documentation.
+[This paragraph](../tutorials/writing-policies/CEL/intro-cel#benefits-of-kubewardens-cel-policy-in-comparison-with-validatingadmissionpolicies)
+explains the benefits of running VAP policies using Kubewarden.
+
+This howto explains how the `kwctl` tool can be used to migrate a VAP policy to Kubewarden.
+
+## Migration steps
+
+:::note
+You must use `kwctl` version 1.14.0 or later to follow this guide.
+:::
+
+Given a file containing this YAML definition of a `ValidatingAdmissionPolicy`:
+
+<details>
+
+<summary>`ValidatingAdmissionPolicy` definition - `vap.yml`</summary>
+
+```yaml
```

Comment:
```suggestion
```yaml title="vap.yml"
```

May be worth instead of having it in the title with the `-`, took me a moment to understand.

Reply:
that's fancy, I didn't know it. I've changed the code to use that both for `vap.yml` and for `vap-binding.yml`